create-and-save-map/src/main/java/com/esri/arcgismaps/sample/createandsavemap/screens/MainScreen.kt

```diff
@@ -0,0 +1,431 @@
+                    sheetContent = {
+                        Column(
+                            Modifier
+                                .padding(12.dp)
+                                .navigationBarsPadding()
+                                .verticalScroll(rememberScrollState())
+                        ) {
+                            CreateMapBottomSheet(
+                                mapViewModel = mapViewModel
+                            )
+
+                            Button(
+                                modifier = Modifier
+                                    .align(Alignment.CenterHorizontally)
+                                    .padding(12.dp),
+                                onClick = {
+                                    composableScope.launch {
+                                        // dismiss bottom sheet immediately
+                                        controlsBottomSheetState.bottomSheetState.hide()
+
+                                        // report success in a snack bar, failure in a popup
+                                        mapViewModel.save().onSuccess {
+                                            snackbarHostState.showSnackbar(
+                                                "Map saved to portal.",
+                                                withDismissAction = true
+                                            )
+                                        }.onFailure { err ->
+                                            mapViewModel.messageDialogVM.showMessageDialog(
+                                                "Error",
+                                                err.message.toString()
+                                            )
+                                        }
+                                    }
+                                }) {
+                                Text("Save to account")
+                            }
+                        }
+                        Spacer(Modifier.size(8.dp))
+                    }
+                ) {}
+            }
+
+            // message dialog can draw over all other content in the main screen
+            mapViewModel.messageDialogVM.apply {
+                if (dialogStatus) {
+                    MessageDialog(
+                        title = messageTitle,
+                        description = messageDescription,
+                        onDismissRequest = ::dismissDialog
+                    )
+                }
+            }
+        }
+    )
+}
+
+/**
+ * Composable function to populate bottom sheet with sample options.
+ */
+@Composable
+fun CreateMapBottomSheet(
+    mapViewModel: MapViewModel
+) {
+
+    Column(
+        modifier = Modifier.padding(vertical = 8.dp), // For spacing top and bottom Composables in this Column
+        verticalArrangement = Arrangement.spacedBy(8.dp) // For spacing between Composables in this Column
```

Comment:
Sorry, these were just placeholder comments 
```suggestion
        modifier = Modifier.padding(vertical = 8.dp),
        verticalArrangement = Arrangement.spacedBy(8.dp)
```